### PR TITLE
Website redesign for ASHG

### PR DESCRIPTION
**Pull request by buske**
This is a PR with Scott's website update.

Resolves the following issues:
* #1 - Improves clarity of MME purpose
* #2 - Includes description of how to get started with MME
* #6 - Imagemap on wheel allows clicking on Matchmaker logos

Most of the differences in index.html are due to whitespace.

### CODE REVIEW

**Comment by buske** [comment]
@Relequestual could you take a look and +1 if all looks okay?

**Comment by Relequestual** [comment]
+1
Comment forthcoming...

File: i_have_a_database.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+          <p>Establish a minimum of two point-to-point API connections to other MME services</p>
+        </li>
+        <li>
+          <p>Contain content that is considered by the MME steering committee to be useful for matching, including the flagging of, or ability to prioritize, candidate genes</p>
+        </li>
+        <li>
+          <p>Successfully implement matching algorithms using test data</p>
+        </li>
+        <li>
+          <p>During user queries, enable dual notification of data requester (i.e. the querier) and prior data depositor (i.e. the queried) including sharing the identities and contact information for each</p>
```
Comment (Relequestual) [inline i_have_a_database.html]: I think we may need to review the wording of this requirement. Currently within Decipher, the data depositor is only notified if the querier decides to follow up on the request (As in, chooses to make contact). In addition, Decipher does not make the identity of the data depositor known. It is up to the original depositor of Decipher data to decide if they wish to responde to any contact request. This is the same way we handle contact requests made by unauthenticated user looking at public consented data on Decipher.

Comment (buske) [inline i_have_a_database.html]: Good comment. I agree. PhenomeCentral is in the same gray area right now. I'm going to go ahead and merge so this is up for the meeting tomorrow, and will move your comment to an issue for us to discuss better wording.

**Pull request merged**